client/galaxy/scripts/mvc/workflow/workflow.js

```diff
@@ -182,17 +182,94 @@ define( [], function() {
                            '<div class="response-message"></div>' +
                            '<h2>Your workflows</h2>' +
                        '</div>'+
-                       '<div class="other-options wf">' +
-                           '<h2>Other options</h2>' +
-                           '<a class="action-button fa fa-cog wf-action" href="'+ Galaxy.root +'workflow/configure_menu" title="Configure your workflow menu">' +
-                               '<span>Configure your workflow menu</span>' +
-                           '</a>' +
-                       '</div>' +
                    '</div>';
         }
     });
 
+    var Run_Workflow_View = Backbone.View.extend({
+
+        initialize: function( options ) {
+            this.setElement( '<div/>' );
+            this.run_workflow( options );
+        },
+
+        /** Open workflow to run */
+        run_workflow: function() {
+            var self = this,
+                id = get_querystring( 'id' ),
+                url = Galaxy.root + 'workflow/run_workflow?id=' + id;
+            $.getJSON(url, function( response ) {
```

Comment:
I suggest to remove the Run_Workflow_View module and just use Utils.get in the show_run function above, and then directly load the ToolForm.View.